Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -4288,7 +4288,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
 #endif
         case CSSPropertyViewTimeline:
         case CSSPropertyViewTimelineInset: // FIXME: view-timeline-inset should be animatable (bug 265690)
-        case CSSPropertyWebkitUserSelect:
+        case CSSPropertyUserSelect:
```

Comment:
This should be moved to between CSSPropertyPage and CSSPropertyScrollMargin.

Reply:
`user-select` should be discretely animatable fwiw: https://drafts.csswg.org/css-ui/#content-selection